src/sdgio-edit.owl

```diff
@@ -1608,6 +1645,291 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynony
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://hdl.handle.net/10947/3865>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.unep.org/sdg/SDGIO_00020315> "A planned process during which an agent - through application or improvement of institutional policies - causes or contributes to increase the capacity of CGIAR partners to take actions concerning agriculture.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020315> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020315> "2019-06-02T20:29:34Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020315> "planned process to apply or improve institutional policies to increase the capacity of CGIAR partners to take actions concerning agriculture"@en)
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020315> <http://purl.unep.org/sdg/SDGIO_00020330>)
+
+# Class: <http://purl.unep.org/sdg/SDGIO_00020316> (planned process to increase the research capacity of CGIAR national partners and beneficiaries)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <https://hdl.handle.net/10947/3865>) <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.unep.org/sdg/SDGIO_00020316> "A planned process during which an agent - through the exchange of information or training -  causes or contributes to increase the research capacity of CGIAR partners.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.unep.org/sdg/SDGIO_00020316> <http://orcid.org/0000-0062-3253-4387>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.unep.org/sdg/SDGIO_00020316> "2019-06-02T20:33:10Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.unep.org/sdg/SDGIO_00020316> "planned process to increase the research capacity of CGIAR national partners and beneficiaries"@en)
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020316> <http://purl.unep.org/sdg/SDGIO_00020331>)
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020316> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002018> <http://purl.unep.org/sdg/SDGIO_00020321>))
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020316> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002018> <http://purl.unep.org/sdg/SDGIO_00020322>))
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020316> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002018> <http://purl.unep.org/sdg/SDGIO_00020323>))
+SubClassOf(<http://purl.unep.org/sdg/SDGIO_00020316> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002018> <http://purl.unep.org/sdg/SDGIO_00020324>))
+
+# Class: <http://purl.unep.org/sdg/SDGIO_00020317> (planned process to improve the forecasting of impacts of climate change)
```

Comment:
Revised label. 

More importantly, this has a circular definition - we can either 1) add climate change forecasting  as a class, cross-link it or 2) define it in this def or 3) both.

I vote for 3).

Reply:
For now I did 2). I agree that 3) would be better. I suppose that forecasting should be a planned process. I will work on that.